metricbeat/README.md

```diff
@@ -50,7 +50,7 @@ See [supported configurations][] for more details.
 * Install it:
   - with Helm 3: `helm install metricbeat elastic/metricbeat`
   - with Helm 2 (deprecated): `helm install --name metricbeat elastic/metricbeat`
-
+  - Add the Elastic Helm charts repo (required for kube-state-metrics chart dependency): `helm repo add stable https://charts.helm.sh/stable`
```

Comment:
NIT but I would put it just before the `helm install` commands because the install will fail if it's not done before.

Reply:
applied the changes